web/Views/Admin/AdminDashboard.cshtml

```diff
@@ -1,4 +1,6 @@
-﻿@model AdminDashboardViewModel
+﻿@using BricksAndHearts.Auth
+@using Microsoft.AspNetCore.Mvc.TagHelpers

```

Comment:
Do we need this? The _ViewImports is importing this :)

Reply:
Yeah it seems we don't need it, I think Rider automatically imports it